SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -349,6 +337,9 @@ TArray<Worker_InterestOverride> USpatialSender::CreateComponentInterest(AActor*
 		FillComponentInterests(SubobjectInfo, bIsNetOwned, ComponentInterest);
 	}
 
+	ComponentInterest.Add({ SpatialConstants::CLIENT_RPC_ENDPOINT_COMPONENT_ID, bIsNetOwned });
```

Comment:
Did we not need to do this previously because there were only commands in the component?

Reply:
Yeah, and the ACLs are reversed with the event-based RPCs. The *sender* has write permission, compared to commands where the *receiver* has write permission.